src/Map/src/UXMapBundle.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\Config\Definition\Configurator\DefinitionConfigurator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\HttpKernel\Bundle\AbstractBundle;
+use Symfony\UX\Map\Bridge as MapBridge;
+use Symfony\UX\Map\Renderer\AbstractRendererFactory;
+use Symfony\UX\Map\Renderer\NullRendererFactory;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class UXMapBundle extends AbstractBundle
+{
+    protected string $extensionAlias = 'ux_map';
+
+    /**
+     * @var array<string, array{ renderer_factory: class-string<AbstractRendererFactory> }>
+     */
+    private static array $bridges = [
+        'google' => ['renderer_factory' => MapBridge\Google\Renderer\GoogleRendererFactory::class],
+        'leaflet' => ['renderer_factory' => MapBridge\Leaflet\Renderer\LeafletRendererFactory::class],
+    ];
```

Comment:
Lets avoid half of the truth source here, the other half in the exception 😅

Reply:
I made this property public but internal, and the `UnsupportedSchemeException` now use it